src/CMakeLists.txt

```diff
@@ -562,27 +581,27 @@ target_sources(QOwnNotes PRIVATE
 
 # The Qt5::Widgets target will also link Qt::Gui and Qt::Core targets
 target_link_libraries(QOwnNotes PRIVATE
-    Qt5::Widgets
-    Qt5::Sql
-    Qt5::Svg
-    Qt5::Network
-    Qt5::Xml
-    Qt5::PrintSupport
-    Qt5::WebSockets
-    Qt5::Qml
-    Qt5::Concurrent
+    Qt${QT_VERSION_MAJOR}::Widgets
+    Qt${QT_VERSION_MAJOR}::Sql
+    Qt${QT_VERSION_MAJOR}::Svg
+    Qt${QT_VERSION_MAJOR}::Network
+    Qt${QT_VERSION_MAJOR}::Xml
+    Qt${QT_VERSION_MAJOR}::PrintSupport
+    Qt${QT_VERSION_MAJOR}::WebSockets
+    Qt${QT_VERSION_MAJOR}::Qml
+    Qt${QT_VERSION_MAJOR}::Concurrent
     QHotkey::QHotkey
     Botan::Botan
     fakevim
     Google::DiffMatchPatch
     md4c-html
 )
-if(UNIX)
-    target_link_libraries(QOwnNotes PRIVATE
-        Qt5::X11Extras
-        ${X11_LIBRARIES}
-    )
-endif()
+# if(UNIX)

```

Comment:
Aren't those needed for global shortcuts? 🤔

Reply:
If it was needed it would fail to link, however its working fine here. Also, .pro file doesn't include this module as well.